bucket/quarto.json

```
@@ -0,0 +1,17 @@
+{
+    "version": "0.9.580",
+    "description": "Quarto is open-source scientific and technical publishing system built on Pandoc.",
+    "homepage": "https://quarto.org",
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/quarto-dev/quarto-cli/releases/download/v0.9.580/quarto-0.9.580-win.msi",
+    "hash": "0901b0570cf3c29138cb56a250062f4942290ea4f255a09bac8c437aac07d8a5",
```

```suggestion
    "architecture": {
        "64bit": {
            "url": "https://github.com/quarto-dev/quarto-cli/releases/download/v0.9.580/quarto-0.9.580-win.msi",
            "hash": "0901b0570cf3c29138cb56a250062f4942290ea4f255a09bac8c437aac07d8a5"
        }
    },
```

The app is built in **64-bit**.

```
> pelook "D:\scoop\apps\quarto\current\Quarto\bin\tools\deno.exe"
loaded "D:\scoop\apps\quarto\current\Quarto\bin\tools\deno.exe" / 61053872 (0x3A39BB0) bytes
signature/type:       PE64 EXE image for amd64
image checksum:       0x03A42076 (OK)
machine:              0x8664 (amd64)
subsystem:            3 (Windows Console)
minimum os:           6.0 (Vista)
linkver:              Microsoft LINK 14.29
```